api/node/rust/interpreter/component_instance.rs

```diff
@@ -137,9 +137,14 @@ impl JsComponentInstance {
             self.inner
                 .set_callback(callback_name.as_str(), {
                     let return_type = return_type.clone();
+                    let callback_name = callback_name.clone();
 
                     move |args| {
-                        let callback: JsFunction = function_ref.get().unwrap();
+                        let Ok(callback) = function_ref.get::<JsFunction>() else {
+                            eprintln!("Ndoe.js: callback {} throws an exception", callback_name);
```

Comment:
- Ndoe.js -> Node.js (as in all the other cases, too)
- Are you sure that this is the place where an exception was thrown? Isn't that in line 151/156? This comment also applies to the other places where `get` on our reference wrapper fails, further down.